selfdrive/controls/lib/latcontrol_pid.py

```diff
@@ -45,4 +45,5 @@ def update(self, active, CS, VM, params, last_actuators, steer_limited, desired_
       pid_log.output = output_steer
       pid_log.saturated = self._check_saturation(self.steer_max - abs(output_steer) < 1e-3, CS, steer_limited)
 
-    return output_steer, angle_steers_des, pid_log
```

Comment:
I didn't set `actuators.steeringAngleDeg` because we already set `pid_log.steeringAngleDesiredDeg`

Reply:
oh this might be why process replay changed